.github/workflows/build_container_image.yml

```diff
@@ -3,43 +3,43 @@ name: Build container image
 on:
   workflow_call:
     inputs:
-      image_name:
-        description: 'Name of the image to build'
-        required: true
-        type: string
-      image_tag_prefix:
-        description: 'Prefix for the image tag'
-        required: false
-        type: string
-        default: ''
-      package_dependencies:
-        description: 'List of dependencies to update'
+      aws_access_key_id:
+        description: 'AWS access key ID'
         type: string
         required: false
-        default: ''
       build_context:
         description: 'Path to the build context'
         type: string
         required: true
-      build_platforms:
-        description: 'Platforms to build the image for'
+      build_platform:
+        description: 'Platform to build the image for'
         type: string
         required: false
-        default: '["linux/amd64","linux/arm64"]'
```

Comment:
Just checking that while the default has changed, we are still building the node-red containers for ARM64

Reply:
It will be defined in the [workflow responsible for building node-red images](https://github.com/FlowFuse/helm/blob/main/.github/workflows/nodered-container.yml#L40). The input name and type are about to change but the whole idea remains unchanged.